raiden-ts/src/path/types.ts

```diff
@@ -68,3 +68,43 @@ export interface RaidenPFS {
   price: BigNumberish;
   token: string;
 }
+
+/**
+ * An IOU used to pay the services
+ */
+export const IOU = t.readonly(
+  t.type({
+    sender: Address,
+    receiver: Address,
+    amount: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    expiration_block: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    one_to_n_address: Address,
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    chain_id: UInt(32),
+    signature: Signature,
+  }),
+);
+
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface IOU extends t.TypeOf<typeof IOU> {}
+
+export interface RaidenIOU {
```

Comment:
I don't see you exposing the `IOU` anywhere. Why the public interface? Also, these public types are only needed if you plan to receive it, not only outputting it, and would need to be exposed in `src/index.ts` for easy access, but in this case I think should remove it.